### DISCUSSION TITLE
pH7-Social-Dating-CMS does not work PERIOD!, unable to login to admin panel.

### DISCUSSION THREAD

**Discussion by Usalabs** [pH7Software/pH7-Social-Dating-CMS#1038] [General]
OK, after downloading the CMS and installing it on my home server, which is fully configured as a web server, running Headless Ubuntu server 20.04 LTS, MySQL, and PHP 7.4.3 with all the extensions needed, and yet after creating a database and installing pH7 CMS, the home page works fine, but the admin login does not work at all, it loops back to the login, the apache error log shows "AH01630: client denied by server configuration: /var/www/html/dating/index.pl", but there is no misconfiguration, it's been checked and rechecked not by me, but a professional technician, and he said it's the pH7 script, because the web server also has e107, PHPFusion, bitweaver and wordpress running and there's not a single error like that, it's only pH7 that's causing the error in the log, so therefore, until it's fixed, I have no choice but to remove it from my server, oh, and don't say, "have you tried this, looked at that, changed this and that", I shouldn't have to do anything but change needed permissions, and I'm not about to ruin the existing configuration that works on all the other PHP scripts, just to satisfy 1 script, if it doesn't work when all other PHP script do, then there's something wrong with the script itself.

**Comment by Picofrank** [comment]
Have you installed the php rewrite module? I had a similar problem (localhosat/admin123 showed a blank page) and this solved it. I'm not a programmer but ph7builder seems to work just fine. As long as you read everything, which is not always easy since information is spread out a bit everywhere. Good luck.

**Reply by Usalabs** [reply]
There is no blank page, and php mod-rewrite is enabled and working otherwise the other php scripts that use .htaccess for re-writes wouldn't work.

I just tried on my friends hosting service, and it too has the exact same problem, the admin login page keeps looping back to login every time I enter the correct login details, and still with the same error "client denied by server configuration:", and his hosting service is a professional service, which means it's the fault of the script NOT the server configuration.

ADDENDUM
I found out, pH7 Social CMS works on XAMMP, but I'm NOT going to change operating systems just to run one script that can easily be fixed by the developer, oh, and I did notice the developer uses XAMPP to test the code, of course it'll work, but try it on a headless Ubuntu 20.04 server then see if it works.
